frontend/javascripts/oxalis/model/helpers/nml_helpers.ts

```diff
@@ -340,16 +340,36 @@ function serializeNodes(nodes: NodeMap): Array<string> {
   });
 }
 
+function getAdditionalCoordinateLabel(useConciseStyle: boolean) {
```

Comment:
Why do we need the non-concise style?

Reply:
this is used in the NMLs. do you think the NMLs should strive for brevity instead of clarity, too?